### PR TITLE
check for server-now via static string

### PR DESCRIPTION
**Pull request by janiswehen**
Closes univelop/univelop#2858

### CODE REVIEW

**Review by philthefox — APPROVED** [review]
as code reviewer

File: lib/src/extensions/server_now.dart [hunk 1]
```diff
@@ -1,5 +1,18 @@
 /// ServerNow is a DateTime extension that returns the current date and time
 /// and uses a FieldValue.serverTimestamp() for the server.
 class ServerNow extends DateTime {
+  /// Is used to identify a ServerNow object in a JSON string.
+  static const String isoString = 'SERVER_NOW_Gr9EpNIetFA3wmC2';
```
Comment (janiswehen) [inline lib/src/extensions/server_now.dart:5]: Server now String

**Review by CLNMR — APPROVED** [review]
Approved

File: lib/src/extensions/server_now.dart [hunk 2]
```diff
@@ -1,5 +1,18 @@
 /// ServerNow is a DateTime extension that returns the current date and time
 /// and uses a FieldValue.serverTimestamp() for the server.
 class ServerNow extends DateTime {
+  /// Is used to identify a ServerNow object in a JSON string.
+  static const String serverNowString = 'SERVER_NOW_Gr9EpNIetFA3wmC2';
+
   ServerNow() : super.now();
```
Comment (janiswehen) [inline lib/src/extensions/server_now.dart:7]: test this.now()

**Pull request merged**